core/src/main/java/org/sonarsource/sonarlint/core/websocket/SonarCloudWebSocket.java

```diff
@@ -111,16 +131,27 @@ private static Optional<? extends ServerEvent> parse(WebSocketEvent event) {
     var eventType = event.event;
     if (eventType == null) {
       return Optional.empty();
-    } else if (!parsersByType.containsKey(eventType)) {
-      SonarLintLogger.get().error("Unknown '{}' event type ", eventType);
-      return Optional.empty();
     }
+
+    return Stream.of(parsersByTypeForProjectFilter, parsersByTypeForProjectUserFilter)
+      .flatMap(map -> map.entrySet().stream())
+      .filter(entry -> eventType.equals(entry.getKey()))
+      .map(Map.Entry::getValue)
+      .findFirst()
```

Comment:
Do we need to do this every time we receive an event? I think we could have a Map field that combines the 2 other Maps